connect_cmd.go

```diff
@@ -64,10 +66,13 @@ func beforeConnectAction(ctx *cli.Context) error {
 	// When machine is already connected, then return error
 	uuid, err := getConsumerUUID()
 	if err != nil {
-		return fmt.Errorf("unable to get consumer UUID: %s", err)
+		return cli.Exit(
+			fmt.Sprintf("unable to get consumer UUID: %s", err),
```

Comment:
I just noticed some inconsistency for when err and err.Error() are used
as part of the arguments to cli.Exit()

In some places, like here, err is used and in others err.Error() is used.

Should err.Error() always be use?

Please confirm they are being used correctly?

Reply:
Using both variants (`fmt.Sprintf("formater %s", err)` or `fmt.Sprintf("formater %s", err.Error())`) is correct. The `error` is interface type, which has `Error()` method returning string.

The documentation https://pkg.go.dev/fmt says: "If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any)."

Reply:
Thank you for confirming @jirihnidek